mozilla/fx-private-relay-add-on
# Set in-page Relay Icon via CSS

## Summary

This PR removes the JavaScript dependency to display the Relay icon on web pages that is injected with the content script.  

## Testing

- Run add-on / log in
- Go to [imgur.com](https://imgur.com/signin?redirect=%2F) (or any page where the icon is expected to be visible)
- Inspect element
- **Expected:** It should be an inline CSS attribute, rather than being pointed at a dedicated add-on resource.
